Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -194,39 +193,6 @@ function Initialize-DevelopmentEnvironment {
     return $script:result | Format-Table
 }
 
-function Invoke-ConfigTransform {
```

Comment:
Note that this file isn't completely migrated yet, [ODS-5378](https://tracker.ed-fi.org/browse/ODS-5378) will complete its migration.
I've just removed references to the deprecated scripts.